INSTRUCTION.md

```diff
@@ -0,0 +1,89 @@
+## Deploy the application following the commands:
+```
+cd ./.infrastructure/  
+kubectl apply -f namespace.yml
+kubectl apply -f busybox.yml
+kubectl apply -f clusterip.yml
+kubectl apply -f nodeport.yml
+kubectl apply -f deployment.yml
+kubectl apply -f hpa.yml
+kubectl apply -f todoapp-pod.yml
+cd ..
+```
+## Check Pod Status:
+```
+kubectl get pods -o wide -n todoapp
```

Comment:
The namespace used here is `todoapp`, but in the deployment and HPA files, the namespace is `mateapp`. Ensure consistency in namespace usage across all commands and configurations.